src/cluster_config.cpp

```diff
@@ -577,6 +577,14 @@ void cass_cluster_set_monitor_reporting_interval(CassCluster* cluster, unsigned
   cluster->config().set_monitor_reporting_interval_secs(interval_secs);
 }
 
+CassError cass_cluster_set_histogram_refresh_interval(CassCluster* cluster, unsigned refresh_interval) {
+  if (refresh_interval <= 0) {
```

Comment:
note: since the parameter is unsigned, we're never going to have a negative interval, and if this is called with -1 they will get a huge interval (arguably this is their fault for not reading the signature)